test/highlight/functions.gleam

```diff
@@ -23,14 +23,14 @@ pub fn replace(
   // ^ type
   //        ^ punctuation.bracket
   string.replace(in: string, each: pattern, with: replacement)
-  // <- variable
+  // <- variable.parameter
   //     ^ property
   //             ^ property
-  //                 ^ variable
+  //                 ^ variable.parameter
   //                         ^ property
-  //                               ^ variable
+  //                               ^ variable.parameter
   //                                        ^ property
-  //                                              ^ variable
+  //                                              ^ variable.parameter
```

Comment:
as an added benefit of `locals.scm` queries, tree-sitter can now track variables through their scope, meaning that these variables which are parameters get highlighted as `variable.parameter` while in scope